cellbase-app/app/docker/README.md

```diff
@@ -0,0 +1,41 @@
+# Building and running Cellbase in Docker
+
+## Build Cellbase
+
+To build cellbase run the follwoing command fromt he root of the repository:

```

Comment:
typo 'fromt'

Reply:
I've not proof read anything, feel free to pick the PR up. Just wanted to get it in there as not going to make progress this week. Feel free to fix the notes up. I actually thought draft would prevent from reviewing it.

Reply:
Nope. Prevents anyone from merging it.